js/framework/AnalyticsConstants.js

```diff
@@ -593,8 +620,10 @@ if (!OO.Analytics.EVENT_DATA)
   /**
    * @public
    * @class Analytics.EVENT_DATA#VideoErrorData
-   * @classdesc Contains information about the error code and message of the video error.
+   * @classdesc (NOTE: will deprecate and be replaced by Analytics.EVENT_DATA.VideoPlaybackErrorData)
+   * Contains information about the error code and message of the video error.
    * @property {string} errorCode The error code
```

Comment:
we should probably put in \@param for the vars used to init the data and \@property for the actual property of the object.